src/rpc/net.cpp

```diff
@@ -146,7 +146,7 @@ static RPCHelpMan getpeerinfo()
                     {
                         {RPCResult::Type::STR, "permission_type", Join(NET_PERMISSIONS_DOC, ",\n") + ".\n"},
                     }},
-                    {RPCResult::Type::NUM, "minfeefilter", /*optional=*/true, "The minimum fee rate for transactions this peer accepts"},
+                    {RPCResult::Type::NUM, "minfeefilter", "The minimum fee rate for transactions this peer accepts"},
```

Comment:
Is is worth it to change the interface, given that there are other optional fields?

Reply:
It maintains the existing interface in releases and follows your comment, "there are some fields that can obviously be made non-optional (such as the fee filter), because they can be absent on the wire even long after the connection has been spun up (thus require a default value anyway)".

Reply:
(Anything that can have an always-present default is simpler for us and for API clients to deal with, code-wise.)